app/models/account.rb

```diff
@@ -109,6 +109,9 @@ def self.columns
 
   before_destroy :destroy_features
 
+  scope :free, -> { where.has { not_exists Contract.where.has { user_account_id == BabySqueel[:accounts].id }.not_paid_since } }
```

Comment:
Can you extract this one? Chaining `where.has` does not really help the reader

Reply:
right 👍 Done 👍